### DISCUSSION TITLE
Dataset with multiple cameras

### DISCUSSION THREAD

**Discussion by ataraxyz** [NVlabs/instant-ngp#826] [Q&A]
Thank you for your excellent work. This iteration of NeRF is so exciting to explore. We appreciate you all!

I have classic Photogrammetry datasets. With drone footage AND high res cameras. Due to the fact that the camera model is set to a single one in colmap this fails. Editing colmap2nerf.py:89 to use multiple cameras ( --ImageReader.single_camera 0 ) lets me generate the transforms, but the resulting nerf model is way less accurate than the model from a single camera source. Surely this is technically possible? Can you help?

**Comment by Tom94** [comment]
Closing as duplicate of https://github.com/NVlabs/instant-ngp/discussions/546
